src/test/lib/operators/join_hash/join_hash_steps_test.cpp

```diff
@@ -80,9 +80,9 @@ TEST_F(JoinHashStepsTest, SmallHashTableAllPositions) {
 
 TEST_F(JoinHashStepsTest, LargeHashTableExistenceOnly) {
   auto table = PosHashTable<int>{JoinHashBuildMode::ExistenceOnly, 100};
-  for (auto i = 0; i < 100; ++i) {
-    table.emplace(i, RowID{ChunkID{ChunkID::base_type{100} + i}, ChunkOffset{200} + i});
-    table.emplace(i, RowID{ChunkID{ChunkID::base_type{100} + i}, ChunkOffset{200} + i + 1});
+  for (auto index = uint16_t{0}; index < 100; ++index) {
```

Comment:
Now that you bring that up, I would even vote for `uint32_t`.
First, `uint8_t` or `uint16_t` do not bring any advantage in memory consumption or performance (I think they are even slower on most CPUs here). I do not remember why I used `uint16_t`. :-/

So, I'd vote for `uint32_t`. Your take on this?

Reply:
> I think they are even slower on most CPUs here

Why?

In the following two lines, `index` is implicitly converted into `uint32_t`, which might be another argument to go for `uint32_t`.

Reply:
> Why?

Because most CPUs work the most efficient when operating on their native word size, which is 64bit for x64 (but I think it's just as fast for 32-bit).